target-app/Onboarding/ViewModels/SignInViewModel.swift

```diff
@@ -13,12 +13,16 @@ protocol SignInViewModelDelegate: AuthViewModelStateDelegate {
 
 class SignInViewModel {
     
-    private var state: AuthViewModelState = .network(state: .idle) {
+    // MARK: - Observed Properties
+    @Published var state: AuthViewModelState = .network(state: .idle) {
         didSet {
             delegate?.didUpdateState(to: state)
         }
```

Comment:
do we need the delegate now? since we are making this a published variable, subscribers will be able to listen for signals instead of becoming the delegate

Reply:
done